spokestack/profile/vad_trigger_asr.py

```diff
@@ -0,0 +1,59 @@
+"""
+Pipeline profile with vad trigger and asr
+"""
+from spokestack.activation_timeout import ActivationTimeout
+from spokestack.asr.speech_recognizer import CloudSpeechRecognizer
+from spokestack.io.pyaudio import PyAudioInput
+from spokestack.pipeline import SpeechPipeline
+from spokestack.vad.webrtc import VoiceActivityDetector, VoiceActivityTrigger
+
+
+class VoiceActivityTriggerASR:
```

Comment:
asr here would probably be better qualified as the spokestack asr, so that you can later add other asr profiles